package.json

```diff
@@ -4,9 +4,10 @@
   "description": "IMPORTANT NOTE This is the first time you will use Gitflow. Keep that in mind! Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise). Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:.Don't make API calls (you'll do it in a following lesson). Use ES6 modules, with import and export.",
   "main": "index.js",
   "scripts": {
-    "start": "webpack serve",
+    "start": "webpack",
```

Comment:
- [ ] I just clone the repo, then I use `npm i` to install everything and then I tried `npm start`, unfortunately, the app doesn't start, I tried reading the readme file if there's a different to proceed but it said using `npm start` is the way to do it. I kindly invite you to change your `package.json` way to start the project. 🐱 
- Hint: You could try `"start": "webpack serve --open",`
![image](https://user-images.githubusercontent.com/20563252/128278053-42d943f5-671b-4dc0-86e1-e7f1206be9cf.png)